raftLogCheck.js

```diff
@@ -0,0 +1,235 @@
+const http = require('http');
+const async = require('async');
+const md5 = require('md5');
+const werelogs = require('werelogs');
+
+const logLevel = Number.parseInt(process.env.DEBUG, 10) === 1
+    ? 'debug' : 'info';
+const loggerConfig = {
+    level: logLevel,
+    dump: 'error',
+};
+werelogs.configure(loggerConfig);
+const log = new werelogs.Logger('s3utils::raftLogCheck');
+
+let pageSize = process.env.PAGE_SIZE;
+if (pageSize === undefined) {
+    pageSize = 1000;
+}
+
+const myArgs = process.argv.slice(2);
+
+if (myArgs.length !== 3) {
+    log.error(
+        'usage: node raftLogCheck.js bucketdHost bucketdPort raftSession');
+    process.exit(1);
+}
+
+const bucketdHost = myArgs[0];
+const bucketdPort = myArgs[1];
+const raftSession = myArgs[2];
+
+function getUrl(url, cb) {
+    log.debug('url', { url });
+    http.get(url, res => {
+        let body = '';
```

Comment:
It could be more efficient to use an array of chunks instead of a string to avoid copying, and at the end, join it with `.join('')`. I think it matters here because returned result could be multi-megabytes and there might be many such calls to check all the logs.